pkgs/development/python-modules/urwid/default.nix

```diff
@@ -68,6 +68,11 @@ buildPythonPackage rec {
 
   pytestFlagsArray = [ "tests" ];
 
+  disabledTests = [
+    # Flaky tests
+    "TwistedEventLoopTest"
```

Comment:
If the test only fails on darwin then why not only disable it on that platform?

Reply:
I suspect this test is flaky on all platforms.